src/js/editor/key-commands.js

```diff
@@ -53,6 +53,26 @@ export const DEFAULT_KEY_COMMANDS = [{
     });
   }
 }, {
+  // FIXME restrict to OS X only?
+  str: 'CTRL+A',
+  run(editor) {
+    let range = editor.cursor.offsets;
+    let {head: {section}} = range;
+    editor.run(postEditor => {
+      postEditor.setRange(new Range(section.headPosition()));
+    });
+  }
+}, {
+  // FIXME restrict to OS X only?
+  str: 'CTRL+E',
```

Comment:
The ctr-A and ctr-E OS X native movement commands don't work when there are atoms in the text (the cursor focus gets moved to the inside of the atom). I have a branch will PR after this that addresses these FIXMEs and scopes to Mac only.